packages/text-area/src/vaadin-text-area-mixin.js

```diff
@@ -178,6 +190,21 @@ export const TextAreaMixin = (superClass) =>
       inputField.scrollTop = scrollTop;
     }
 
+    /** @private */
+    __updateMinHeight(minRows) {
+      if (!this._inputField || !this.inputElement) {
+        return;
+      }
+
+      // For minimum height, just set the number of rows on the native textarea,
+      // which causes the input container to grow as well.
+      // Do not override this on custom slotted textarea as number of rows may
+      // have been configured there.
+      if (this.inputElement === this.__textAreaController.defaultNode) {
+        this.inputElement.rows = Math.max(minRows, 2);
```

Comment:
Shrinking either the Lumo or Material component to a single row, so that it aligns with other components like text field, is not possible without additional style changes which don't seem trivial. I'd suggest to exclude this for now and discuss separately how we can solve this.

Reply:
Maybe we could add a warning when `minRows` is less than 1 since some users might think it's a bug.

Reply:
Added a warning.